images/01-base/Dockerfile

```diff
@@ -35,6 +35,9 @@ COPY inputrc /etc/inputrc
 COPY growpart /usr/bin/growpart
 RUN sed -i s/"partx --update \"\$part\" \"\$dev\""/"partx --update --nr \"\$part\" \"\$dev\""/g /usr/bin/growpart && \
     sed -i -e 's/duid/clientid/g' /etc/dhcpcd.conf && \
-    sed -i 1,10d /etc/rsyslog.conf
+    sed -i 1,10d /etc/rsyslog.conf && \
+    echo "*.*                /var/log/syslog" >> /etc/rsyslog.conf
```

Comment:
this is basically a catchall to log to file anything that we're otherwise not logging.

like the default to log debug to dmesg, I'm not yet sure if it'll stay in 1.0 - it depends on what we see getting logged.